apko-build/action.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'Build image with apko'
+description: |
+  This action build an OCI image with apko, given a config file
+  and tag to use.
+
+inputs:
+  config:
+    description: |
+      The config file to use for building the image.
+    default: .apko.yaml
+
+  tag:
+    description: |
+      The tag to use for publishing the image.
+    required: true
+
+  repository_owner:
+    description: |
+      The repository owner's GitHub username.
+    default: ${{ github.repository_owner }}
+
+  token:
+    description: |
+      The repository owner's GitHub token.
+    default: ${{ github.token }}
+
+runs:
+  using: docker
+  image: 'docker://ghcr.io/chainguard-dev/apko:v0.1'
```

Comment:
... or I'm guessing the container this runs in creates trouble for the above, which is what you were saying in slack 🤔

Reply:
I'm not sure, I can try the docker login action, but I'm pretty sure it wasn't being copied into the container.